components/admin_console/admin_definition.jsx

```diff
@@ -513,13 +513,13 @@ export default {
                             label: 'admin.log.enableWebhookDebugging',
                             label_default: 'Enable Webhook Debugging:',
                             help_text: 'admin.log.enableWebhookDebuggingDescription',
-                            help_text_default: 'To output the request body of incoming webhooks to the console, enable this setting and set {boldedConsoleLogLevel} to "DEBUG". Disable this setting to remove webhook request body information from console logs when in DEBUG mode.',
+                            help_text_default: 'When true, sends webhook debug messages to the server logs. To also output the request body of incoming webhooks, set {boldedLogLevel} to "DEBUG".',
                             help_text_values: {
-                                boldedConsoleLogLevel: (
+                                boldedLogLevel: (
                                     <strong>
                                         <FormattedMessage
-                                            id='admin.log.consoleLogLevel'
-                                            defaultMessage='Console Log Level'
+                                            id='admin.log.fileLogLevel'
```

Comment:
Id doesn't match now, should be `admin.log.logLevel`

Reply:
@jwilander Sorry - this is a bit embarrassing..thanks for the catch. I reviewed the diff file line by line,